packages/react-native/ReactCommon/react/renderer/components/view/platform/macos/react/renderer/components/view/HostPlatformViewProps.cpp

```diff
@@ -47,19 +47,37 @@ HostPlatformViewProps::HostPlatformViewProps(
                     rawProps,
                     "enableFocusRing",
                     sourceProps.enableFocusRing,
+                    {})),
+      keyDownEvents(
+          ReactNativeFeatureFlags::enableCppPropsIteratorSetter()
+              ? sourceProps.keyDownEvents
+              : convertRawProp(
+                    context,
+                    rawProps,
+                    "keyDownEvents",
+                    sourceProps.keyDownEvents,
+                    {})),
+      keyUpEvents(
+          ReactNativeFeatureFlags::enableCppPropsIteratorSetter()
+              ? sourceProps.keyUpEvents
+              : convertRawProp(
+                    context,
+                    rawProps,
+                    "validKeysUp",
```

Comment:
Why is this `validKeysUp`?
```suggestion
                    "keyUpEvents",
```